src/scraper/__main__.py

```diff
@@ -196,117 +380,33 @@ def scrape(self):
-                            url=urllib.parse.urljoin(base_url, "Case/CaseDetail"),
-                            verification_text="Case Information",
-                            logger=self.logger,
-                            ms_wait=args.ms_wait,
-                            params={
-                                "eid": case_json["EncryptedCaseId"],
-                                "CaseNumber": case_json["CaseNumber"],
-                            },
-                        )
-                        # make request for financial info
-                        case_html += request_page_with_retry(
-                            session=self.session,
-                            url=urllib.parse.urljoin(
-                                base_url, "Case/CaseDetail/LoadFinancialInformation"
-                            ),
-                            verification_text="Financial",
-                            logger=self.logger,
-                            ms_wait=args.ms_wait,
-                            params={
-                                "caseId": case_json["CaseId"],
-                            },
-                        )
-                        # write case html data
-                        self.logger.info(f"{len(case_html)} response string length")
-                        with open(
-                            os.path.join(self.case_html_path, f"{case_id}.html"), "w"
-                        ) as file_handle:
-                            file_handle.write(case_html)
-                        if case_id not in cached_case_list:
-                            cached_case_list.append(case_id)
-                        if args.test:
-                            self.logger.info("Testing, stopping after first case")
-                            sys.exit()
+                    self.scrape_case_data_post2017(cached_case_list, base_url)
 
-        self.logger.info(f"\nTime to run script: {round(time() - START_TIME, 2)} seconds")
+    def scrape(self):
+        self.base_url, self.odyssey_version, self.notes = self.get_ody_link(self.county)
+        self.main_page_html, self.main_soup = self.scrape_main_page(self.base_url, self.odyssey_version, self.notes)
+        self.search_url, self.search_page_html, self.search_soup = self.scrape_search_page(self.base_url, self.odyssey_version, self.main_page_html, self.main_soup)
+        self.hidden_values = self.scrape_hidden_values(self.odyssey_version, self.main_soup, self.search_soup)
+        if self.case_number:
+            self.scrape_individual_case(self.base_url, self.search_url, self.hidden_values, self.case_number)
+        else: # scrape a list of JOs between a start and end date
+            self.judicial_officers, self.judicial_officer_to_ID = self.scrape_jo_list(self.odyssey_version, self.search_soup, self. judicial_officers)
+            self.START_TIME = time() # initialize variables to time script and build a list of already scraped cases
```

Comment:
🔧 May want to rename this `SCRAPE_START_TIME`, or something a little more specific.

Reply:
It might also just be fine as a local variable in this method, rather than an instance variable

Reply:
I'll take a look to double check if it is used anywhere else or if it's local and then update the field name accordingly.